jjb/clm/templates/clm-go-templates.yaml

```diff
@@ -0,0 +1,189 @@
+####################
+# COMMON FUNCTIONS #
+####################
+
+- lf_go_common: &lf_go_common
```

Comment:
We can actually remove this lf_go_common block and just re-use "lf_maven_common" . I noticed they are same.